### PR TITLE
entr: add test and modernize

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: Library/Formula/entr.rb [hunk 1]
```diff
@@ -13,10 +11,19 @@ class Entr < Formula
...
     system "make"
     system "make install"
   end
+
+  test do
+    touch testpath/"test.1"
+    fork do
+      sleep 0.5
+      touch testpath/"test.2"
+    end
```
Comment (mikemcquaid) [inline Library/Formula/entr.rb]: Why does this need a fork, out of interest?

Comment (xu-cheng) [inline Library/Formula/entr.rb]: This is a program to monitor file changing. So fork a subprocess to change the file.

File: Library/Formula/entr.rb [hunk 2]
```diff
@@ -13,10 +11,19 @@ class Entr < Formula
   end
 
   def install
-    ENV['PREFIX'] = prefix
-    ENV['MANPREFIX'] = man
+    ENV["PREFIX"] = prefix
+    ENV["MANPREFIX"] = man
     system "./configure"
     system "make"
     system "make install"
```
Comment (xu-cheng) [inline Library/Formula/entr.rb]: Ops, forget to modernize this one.

Comment (mikemcquaid) [inline Library/Formula/entr.rb]: :+1:

File: Library/Formula/entr.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/entr.rb]: Cool :+1:

**Pull request closed without merging**